library/build.gradle

```diff
@@ -8,7 +8,7 @@ android {
     compileSdkVersion rootProject.compileSdkVersion
 
     compileOptions {
-        kotlinOptions.freeCompilerArgs += ['-module-name', "com.github.ChuckerTeam.Chucker.library-no-op"]
```

Comment:
++

Reply:
Unfamiliar with what's going on here. Can someone explain?

Reply:
Sure. This was a fix I introduced for #134 (in #146). I accidentally copy and pasted the same line for `library` and `library-no-op` without adapting the module name to the proper artifact name.